flutter-studio/src/io/flutter/actions/FlutterNewProjectAction.java

```diff
@@ -41,7 +41,7 @@ public void update(@NotNull AnActionEvent e) {
     if (NewWelcomeScreen.isNewWelcomeScreen(e)) {
       e.getPresentation().setIcon(getFlutterDecoratedIcon());
       e.getPresentation().setText(
-        Registry.is("use.tabbed.welcome.screen")
+        Registry.is("use.tabbed.welcome.screen", false)
```

Comment:
Is this a related change?

Reply:
It fixes a problem when the item is not found.